summerfi-api/get-morpho-claims-function/src/get-claims.ts

```diff
@@ -45,14 +47,22 @@ const aggregateClaimsByToken = (claims: MorphoAggregatedClaims[]): MorphoAggrega
   return Object.values(aggregatedClaims)
 }
 
-export const getClaims = async ({ account, logger }: GetClaimsParams): Promise<MorphoClaims> => {
+export const getClaims = async ({
+  account,
+  chainId,
+  claimType,
+  logger,
+}: GetClaimsParams): Promise<MorphoClaims> => {
   try {
+    if (![ChainId.MAINNET, ChainId.BASE].includes(chainId)) {
+      throw new Error(`Not supported chain id ${chainId}. Supported chain ids are: 1, 8453.`)
+    }
+
     const [responseRewards, responseDistribution] = await Promise.all([
-      fetch(`https://rewards.morpho.org/v1/users/${account}/rewards`),
-      fetch(`https://rewards.morpho.org/v1/users/${account}/distributions`),
+      fetch(`https://rewards.morpho.org/v1/users/${account}/rewards?chain_id=${chainId}`),
+      fetch(`https://rewards.morpho.org/v1/users/${account}/distributions?chain_id=${chainId}`),
     ])
```

Comment:
what about error handling?

Reply:
it is inside try catch

Reply:
I mean shouldn't client be informed there was an error?   there is only console log right now

Reply:
looks good